datalad/core/distributed/tests/test_clone.py

```diff
@@ -741,6 +741,19 @@ def test_relative_submodule_url(path=None):
         '../../origin')
 
 
+def test_mushy_windows_submodule_url(tmp_path):
+    """Test that subdataset clones from relative urls
+    do not include backslashes (gh-7180)"""
+    Dataset(tmp_path / 'origin').create()
+    ds = Dataset(tmp_path / 'ds').create()
+    with chpwd(ds.path):
+        ds_cloned = ds.clone(
+            source=op.join(op.pardir, 'origin'),
+            path='sources')
+        assert '\\' not in ds_cloned.config.get('remote.dl-test-remote.url')
```

Comment:
`dl-test-remote` is a defined constant. Please use it instead of hardcoding names in tests.

Reply:
thx for catching that. I wanted to squash the fix into [6d1d129](https://github.com/datalad/datalad/pull/7181/commits/6d1d129280bc99590fe7a11c4981739ccb087102), but that messed up the merge commit in between - so now its a standalone commit [305a256](https://github.com/datalad/datalad/pull/7181/commits/305a256e1a2db1d7d8af59d3f5e4200012070c7f)